demo.html

```diff
@@ -4,10 +4,15 @@
 <head>
   <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes">
   <title>Google Map demo</title>
-  <script src="../webcomponentsjs/webcomponents.min.js"></script>
+  <script src="../webcomponentsjs/webcomponents.js"></script>
   <link rel="import" href="google-map.html">
+  <link rel="import" href="google-map-marker.html">
```

Comment:
Several reasons:
- you might want to use google-map without markers, why download extra code
- 1 component per file is (mostly) the Polymer way
- google-map is 700+ lines, google-map-marker 400, having them both in same file made understanding harder